project-reports/2023/2023-Q3-Hyperledger-FireFly.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: default
+title: 2023 Q3 Hyperledger FireFly
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# Project Health
+
+Hyperledger FireFly is applying to move to Graduated Project status. A separate document will be submitted to the TOC to propose this change. Throughout the past year, there has been very strong activity within the project, and Hyperledger FireFly has continued to gain even more momentum this quarter. FireFly is powering many use cases in the blockchain and digital assets community, with recent examples that include Swift’s phase 2 CBDC work with 32 central and global transaction banks across 4 use cases, and the DFCRC and Reserve Bank of Australia CBDC work running 15 projects including ones with ANZ Bank and Mastercard.
```

Comment:
Please do create an issue in the [hyperledger/toc repo](https://github.com/hyperledger/toc/issues) entitled "Move FireFly to Graduated Status" so that we can ensure that this gets added to a future meeting for discussion.

Reply:
Thanks Tracy! I am working on this in parallel. Should it just be an issue here, or should it be a PR raised to https://github.com/hyperledger/hyperledger-hip ?

Reply:
@nguyer we clarified on the TOC call that it should be written as an issue in the TOC repo. The Cacti team has started one here: https://github.com/hyperledger/toc/issues/164 for their application

Reply:
Thanks for the clarifications! I will open an issue the same way. I don't have slides, but I do have a lot of Markdown I will post :)